thewalrus/quantum/conversions.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2019-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Functions for transforming one type of covariance-matrix-like object into
+another as well as reducing a Gaussian state.
```

Comment:
Nice! Should this module be named `covariance.py`?

Reply:
Discussed this with @nquesada and we decided to go for `conversions` since there are also functions that convert means, not only covariances. I think this is quite clear when looking at the full name `thewalrus.quantum.conversions`, but I'm open for change. :slightly_smiling_face: